src/sentry/integrations/gitlab/search.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.models import Integration
+
+
+class GitlabIssueSearchEndpoint(OrganizationEndpoint):
+    permission_classes = (OrganizationPermission, )
+
+    def get(self, request, organization, integration_id):
+        try:
+            integration = Integration.objects.get(
+                organizations=organization,
+                id=integration_id,
+                provider='gitlab',
+            )
+        except Integration.DoesNotExist:
+            return Response(status=404)
+
+        field = request.GET.get('field')
+        query = request.GET.get('query')
+        if field is None:
+            return Response({'detail': 'field is a required parameter'}, status=400)
+        if not query:
```

Comment:
Actually... query probably needs to be `if query is None` or `if query == '' or query is None` rather than `if not query`. What if you literally have something you're searching for and you enter a `0`? weird edge case, but I see no reason why this wouldn't be a valid search. @markstory what do you think?